schema.sql

```diff
@@ -0,0 +1,63 @@
+--
+-- PostgreSQL database dump
+--
+
+-- Dumped from database version 11.2 (Debian 11.2-1.pgdg90+1)
+-- Dumped by pg_dump version 11.2 (Debian 11.2-1.pgdg90+1)
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET idle_in_transaction_session_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+SET row_security = off;
+
+--
+-- Name: user_auth; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA user_auth;
+
+
+ALTER SCHEMA user_auth OWNER TO postgres;
+
+--
+-- Name: auth_id_seq; Type: SEQUENCE; Schema: user_auth; Owner: postgres
+--
+
+CREATE SEQUENCE user_auth.auth_id_seq
```

Comment:
You shouldn't need to create your own sequence unless it has something special on it

Reply:
Is there just a stock incrementing sequence to use in cases like this? Is that making id `serial` type will do?

Reply:
Yup, that would be correct